### ISSUE TITLE
fields.json repeatble nodes issue

### ISSUE THREAD

**Issue by karel-debrabandere-bosa-fgov-be** [OP-TED/eForms-SDK#360]
## Problem:
When we use our notice rendering engine, some conceptual node is rendered as two separate xml nodes (`<cac:CallForTendersDocumentReference>`), but it according to the XSDs it should be a single one.

```
<cac:TenderingTerms>
  <!-- ... -->
  <cac:CallForTendersDocumentReference>
    <cbc:ID>TenderDocumentRef-LOT-0000</cbc:ID>
    <cac:Attachment>
      <cac:ExternalReference>
        <cbc:URI>https://tst.publicprocurement.be/supplier/enterprises/0/tendering-workspaces/publication-workspace-detail/9352d0d7-a9a9-4f2d-83e3-9349184e3751/documents</cbc:URI>
      </cac:ExternalReference>
    </cac:Attachment>
  </cac:CallForTendersDocumentReference>
  <!-- ... -->
</cac:TenderingTerms>
```

```
<cac:TenderingTerms>
  <!-- ... -->
  <cac:CallForTendersDocumentReference>
    <cbc:ID>TenderDocumentRef-LOT-0000</cbc:ID>
  </cac:CallForTendersDocumentReference><!-- should not be here! -->
  <cac:CallForTendersDocumentReference><!-- should not be here! -->
    <cac:Attachment>
      <cac:ExternalReference>
        <cbc:URI>https://tst.publicprocurement.be/supplier/enterprises/0/tendering-workspaces/publication-workspace-detail/9352d0d7-a9a9-4f2d-83e3-9349184e3751/documents</cbc:URI>
      </cac:ExternalReference>
    </cac:Attachment>
  </cac:CallForTendersDocumentReference>
  <!-- ... -->
</cac:TenderingTerms>
```

## Investigation

Our conceptual model looks as follows:
```
{
  "nodeId": "ND-LotTenderingTerms",
  "children": [
    {
      "fieldId": "BT-744-Lot",
      "parentId": "ND-LotTenderingTerms",
      "value": "false"
    },
    {
      "fieldId": "BT-97-Lot",
      "parentId": "ND-LotTenderingTerms",
      "value": "ENG"
    },
    {
      "nodeId": "ND-LotProcurementDocument",
      "children": [
        {
          "fieldId": "OPT-140-Lot",
          "parentId": "ND-LotProcurementDocument",
          "value": "TenderDocumentRef-LOT-0003"
        }
      ]
    },
    {
      "nodeId": "ND-TenderRecipient",
      "children": [
        {
          "fieldId": "BT-18-Lot",
          "parentId": "ND-TenderRecipient",
          "value": "test"
        }
      ]
    },
    {
      "nodeId": "ND-LotDocumentsReference",
      "children": [
        {
          "fieldId": "BT-15-Lot",
          "parentId": "ND-LotDocumentsReference",
          "value": "https://tst.test.be/"
        }
      ]
    }
  ]
}
```

This is build based on the fields.json metadata:

```
{
  "id" : "ND-LotProcurementDocument",
  "parentId" : "ND-LotTenderingTerms",
  "xpathAbsolute" : "/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:TenderingTerms/cac:CallForTendersDocumentReference",
  "xpathRelative" : "cac:CallForTendersDocumentReference",
  "repeatable" : true
},
{
  "id" : "ND-LotDocumentsReference",
  "parentId" : "ND-LotTenderingTerms",
  "xpathAbsolute" : "/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:TenderingTerms/cac:CallForTendersDocumentReference[not(cbc:DocumentType/text()='restricted-document')]",
  "xpathRelative" : "cac:CallForTendersDocumentReference[not(cbc:DocumentType/text()='restricted-document')]",
  "repeatable" : true
},

// ...

{
  "id" : "OPT-140-Lot",
  "parentNodeId" : "ND-LotProcurementDocument",
  "name" : "Procurement Documents ID",
  // ...
},
{
  "id" : "BT-15-Lot",
  "parentNodeId" : "ND-LotDocumentsReference",
  "name" : "Documents URL",
  // ...
}
```

Note that the fields `OPT-140-Lot` and `BT-15-Lot` are children of the nodes `ND-LotProcurementDocument` and `ND-LotDocumentsReference`, respectively. Both nodes are repeatable independent nodes under `ND-LotTenderingTerms`. 

## Issue
We would expect that these fields to be part of a single repeatable conceptual subtree instead of being independently repeatable in the conceptual model. As far as we can tell this is already the case in the visual metadata model and in the XSDs.

**Comment by DeMol-EE** [comment]
We made the following changes to fields.json to test our hypothesis:

```diff
   "id" : "ND-LotDocumentsReference",
-  "parentId" : "ND-LotTenderingTerms",
+  "parentId" : "ND-LotProcurementDocument",
   "xpathAbsolute" : "/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:TenderingTerms/cac:CallForTendersDocumentReference[not(cbc:DocumentType/text()='restricted-document')]",
   "xpathRelative" : "cac:CallForTendersDocumentReference[not(cbc:DocumentType/text()='restricted-document')]",
-  "repeatable" : true
+  "repeatable": false
```

These changes resolve our issue. Could you please verify if the metadata in the SDK is correct?

**Comment by bertrand-lorentz** [comment]
This will be resolved in version 1.7.0 of the SDK (see linked commit).
So I'm closing this.
